ecms_base/ecms_base.install

```diff
@@ -448,6 +448,19 @@ function ecms_base_update_9029(array &$sandbox): void {
  */
 function ecms_base_update_9030(array &$sandbox): void {
 
+  // Install the newly introduced purge modules.
+  $modules_to_install = [
```

Comment:
Might need a rebase? I added the `ecms_publications` to the module install list too.

Reply:
Yep, I think everything's in place now